packages/babel-preset-env/README.md

```diff
@@ -18,22 +18,20 @@ Without any configuration options, @babel/preset-env behaves exactly the same as
 
 You can also configure it to only include the polyfills and transforms needed for the browsers you support. Compiling only what's needed can make your bundles smaller and your life easier.
 
-This example only includes the polyfills and code transforms needed for coverage of users > 0.25%, ignoring Internet Explorer 11 and Opera Mini. We use [browserslist](https://github.com/ai/browserslist) to parse this information, so you can use [any valid query format supported by browserslist](https://github.com/ai/browserslist#queries).
+By default, it includes the polyfills and code transforms needed for [this modern browsers](http://browserl.ist/?q=defaults).
 
-```js
-{
-  "presets": [
-    ["@babel/preset-env", {
-      "targets": {
-        // The % refers to the global coverage of users from browserslist
-        "browsers": [ ">0.25%", "not dead"]
-      }
-    }]
-  ]
-}
+If you want to specify own target browsers, we recommend to use [`.browserslistrc`](https://github.com/browserslist/browserslist) config. This config is used by many tools including Autoprefixer.
+
+This example only includes the polyfills and code transforms needed for coverage of users > 0.25%, ignoring browsers without security updates like IE Internet Explorer 10 and BlackBerry.
+
+```
+> 0.25%
+not dead
 ```
 
-> You can also target individual versions of browsers instead of using a query with `"targets": { "chrome": "66" }`.
+The full list of queries could be found in [Browserslist docs](https://github.com/browserslist/browserslist#queries).
+
+If you need to use different browssers for Autoprefixer and Babel, you can use `browsers` options of each tool.
```

Comment:
Also, after `you can use browser options of each tool`: we could have [removed JSON `targets.browsers` configuration example](https://github.com/babel/babel/pull/7835/files#diff-da1de098f27287f389b3b8f85303e4f1L23) from above.

Reply:
I remove it by design. Most users don’t read the text. They just quick look for example and copy-paste them to try, rather than read all warnings and explanation.

There is a good rule for frameworks. Dangerous things must be harder to implement, rather than safer things. Remember how React is using long options (`dangerouslySetInnerHTML`) to force people to avoid them.

Same logic here. If you really want to use `browsers` (there are only a few cases, like bundlers) you are capable enough to understand docs (not everyone creates bundlers). For others, it will be a good limit to use best practices.